### ISSUE TITLE
Changing Origin of Drawables not working

### ISSUE THREAD

**Issue by tgb20** [TurtleP/LovePotion#49]
When changing the origin of a drawable the change is not reflected in the switch build. This can be observed when using ```love.graphics.draw``` and changing the rotation origin, or when using ```love.graphics.printf``` and using "center".

When running locally this change is applied, but after exporting the .nro the origin is reset.

**Comment by TurtleP** [comment]
I’ll make a commit with the update later today and see about a new release shortly after, assuming more graphics bugs don’t show up :p

**Comment by tgb20** [comment]
Any ETA on this update?

**Comment by TurtleP** [comment]
Sorry I’ve been a bit busy. Tried getting some extra nice upgrades in and it didn’t work out. 

I’ll fix it and make the commit tomorrow for sure. Today’s thanksgiving, and I would have done it if I had the time 😅

**Comment by TurtleP** [comment]
3 days later. . finally made the commit. Sorry for the delay! Everything should be fine now, still needs some other tests before I push 1.0.3.1.

**Comment by RattletraPM** [comment]
Sadly this still doesn't seem to work. I tested it using LÖVE 11.1 on my PC (running Debian Buster) and a few LovePotion nightlies (1.0.3.11, 1.0.3.10 and 1.0.3.9) on my Switch. I didn't test this on a 3DS.

Setting different origin offsets using love.graphics.draw on my PC works fine, however it gets reset at 0,0 on LovePotion for the Switch.

**Comment by TurtleP** [comment]
iirc [@Sam Breadman](https://github.com/samthebreadman) has some fixes, and one of them includes this. I've been poking them for the code for a while now. Hoping to get it soon 👀

**Comment by RattletraPM** [comment]
Great, thanks for the heads up! :smile: 

I imagined that love.graphics.rotate wasn't implemented yet but I wanted to report it just to be sure. That aside I'll probably continue to use the workaround as I'd prefer to use stable versions rather than nightlies for my game but it's good to know that it'll eventually get fixed, especially if a new stable will get released soon.

In the meantime I figured I'd post the code for the workaround if anyone else feels like I do, so [here it is](https://pastebin.com/6Net9PvK)! For reference, it's been used in [this demo](https://studio.youtube.com/video/yKbDu0qK2us/edit) (even if it could be a bit hard to see due to the effect used).
